.github/workflows/ci.yml

```diff
@@ -204,6 +204,56 @@ jobs:
         run: |
           make test-cucumber
 
+  e2e-containerd-docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      -
+        name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+          check-latest: true
+          cache: true
+      -
+        name: Setup docker CLI
+        run: |
+          curl https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_CLI_VERSION}.tgz | tar xz
+          sudo cp ./docker/docker /usr/bin/ && rm -rf docker && docker version
```

Comment:
This will be overridden by `crazy-max/ghaction-setup-docker` (24.0.6). See https://github.com/docker/compose/actions/runs/6338599258/job/17216038445?pr=11046#step:7:261